requirements.txt

```diff
@@ -1,6 +1,2 @@
-fontmake>=2.4
-fontbakery>=0.8.11
-gftools[qa]>=0.9.23
-drawbot-skia>=0.4.8
-sh>=1.14.2
-bumpfontversion>=0.2.0
+# Placeholder file, update the requirements by running `make update`.
+-r requirements.in
```

Comment:
is this "requirements.txt" file even needed in the template repo, now that it's going to be auto-regenerated?

Reply:
I guess it's needed for when you `make build` without updating first.

Reply:
LMGTFY https://pip-tools.readthedocs.io/en/latest/#a-note-on-resolvers